src/Api/WebSocket/Types.hs

```diff
@@ -50,6 +52,7 @@ data Method
   | GetBlock
   | StartFetchBlocks Int64 Text DatumFilter
   | CancelFetchBlocks
+  | GetHealthcheck
```

Comment:
I think `HealthCheck` might be a clearer name, but that's just my personal preference